### PR TITLE
Optimize no substitution case

### PR DESCRIPTION
**Pull request by cyx**
For cases when the localizations are simple strings, no need to incur the overhead of doing a `template.New + template.Parse + template.Execute`.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/m1/go-localize
+module github.com/cyx/go-localize
```
Comment (m1) [inline go.mod]: Why was this changed?

**Review by m1 — CHANGES_REQUESTED** [review]
Please update readme to document changes and add/update tests!

File: go.mod (hunk 1 above)
Comment (cyx) [reply go.mod]: Just needed something for our project so we can move forward. I've put it back so we can consider merging.

**Comment by cyx** [comment]
> Please update readme to document changes and add/update tests!

There are actually no behavior changes worth updating the README for, nor are there any tests that I think are worth it as well. If you have ideas, let me know though -- (e.g. really the change is internal only and so existing tests should work the same)
